build/Targets/Packages.props

```diff
@@ -80,7 +80,7 @@
     <MicrosoftNETCorePortableCompatibilityVersion>1.0.1</MicrosoftNETCorePortableCompatibilityVersion>
     <MicrosoftNETCoreRuntimeCoreCLRVersion>2.0.0</MicrosoftNETCoreRuntimeCoreCLRVersion>
     <MicrosoftNETCoreTestHostVersion>1.1.0</MicrosoftNETCoreTestHostVersion>
-    <MicrosoftNetFX20Version>1.0.3</MicrosoftNetFX20Version>
+    <MicrosoftNetFrameworkReferenceAssembliesVersion>1.0.0-alpha-004</MicrosoftNetFrameworkReferenceAssembliesVersion>
```

Comment:
Shouldn't we also be deleting the old package we used to use?

Reply:
Deleting the code that produces that produces that package? I think I already did that before :)

Reply:
No I meant deleting the package that brings in the reference assemblies that I created some time ago. That is referenced here and it would seem this supercedes it now: RoslynToolsReferenceAssembliesVersion

Reply:
Oh, that thing. Yeah, I'll delete that.

Reply:
> that thing

that glorious awful hack 😄